### PR TITLE
🚀 Release ➾ Taqueria v0.24.1

### PR DESCRIPTION
**Pull request by hu3man**
# 🌮 Taqueria PR

## 🪁 Description

This PR covers the version bump and required updates to documentation and release notes for the Taqueria v0.24.0 release

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [x] ⛱️ I have completed this PR template in full and updated the title appropriately
- [x] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [x] 🤿 Corresponding changes have been made to all documentation
- [ ] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

## 🎢 Test Plan

Manual verification via release checklist

## 🛸 Type of Change

- [ ] 🧽 Chore ➾
- [ ] 🛠️ Fix ➾
- [ ] ✨ Feature ➾
- [ ] 👷 Refactor ➾
- [ ] 🧪 Pre-Release ➾
- [x] 🚀 Release ➾

### CODE REVIEW

File: website/docs/release-notes.mdx [hunk 1]
```diff
@@ -2,6 +2,166 @@
...
+
+| Details     |               |
+| ----------- | ------------- |
+|Release Date | Nov 04, 2022  |
+|Release Type | Minor         |
+|Release Page | [v0.24.0](https://github.com/ecadlabs/taqueria/releases/tag/v0.24.0) |
+
+### Summary of Impactful Changes
+
+- A new plugin `@taqueria/plugin-core` has been added. It provides a task `taq clean` which removes stale docker images
```
Comment (AlirezaHaghshenas) [inline website/docs/release-notes.mdx:15]: Maybe adding that it gets installed with `init`?

**Review by AlirezaHaghshenas — APPROVED** [review]

**Review by jchenche — APPROVED** [review]
Let's ship it!

**Comment by AlirezaHaghshenas** [comment]
Do we need to inform users that they need to manually install the core plugin after upgrading Taqueria?
Or do we prefer to never put an essential command in it?

**Comment by jchenche** [comment]
@AlirezaHaghshenas I think currently users have to manually install all the plugins after upgrading Taqueria no? Alternatively, run the `jq` script that I believe was made for this? @hu3man right?

**Pull request merged**